package.json

```diff
@@ -40,9 +44,9 @@
   },
   "dependencies": {
     "@grpc/grpc-js": "1.3.7",
-    "@momento/wire-types-javascript": "0.8.0",
     "jwt-decode": "3.1.2",
-    "toml": "^3.0.0"
+    "toml": "^3.0.0",
+    "wire-types-javascript": "0.9.0"
```

Comment:
Dang. I think this line really ought to be called momento-wire-types. Maybe this package should be momento-sdk?

Reply:
I can rename the wire types package to `momento-wire-types` in [this pr](https://github.com/momentohq/client_protos/pull/36). 
I will bring up the name for the JS sdk package in the developer-ecosystem channel.